docs/Manual/04 Fusing.md

```diff
@@ -116,55 +116,57 @@ for i in range(0, 16, 4):
             for jj in range(4):
                 C[i+ii, j+jj] *= B[i+ii, j+jj]
 ```
+## Constraints of Fusing Dimension
+The fusing dimension comes with certain constraints that are discussed from the `safety` perspective with examples. 
 
 ### Constraint 1: the fusing dimension is executed sequentially
-The fusing dimension has a special constraint, which does not apply to other dimensions. Specifically, the fusing dimension cannot be parallelized, vectorized, or tensorized (see [Section 7](<07%20Plans%20-%20Vectorization%20and%20Parallelization.md>) ) and it must be executed sequentially. This constraint enables the safety guarantee discussed below.
+Unlike other dimensions that allow parallelization and vectorization (parallelization and vectorization are presented in [Section 7](<07%20Plans%20-%20Vectorization%20and%20Parallelization.md>) ), the fusing dimension can neither be parallelized nor vectorized. It has an inherent execution constraint and must be executed sequentially. However, this sequential execution constraint allows the safety guarantee discussed below.   
```

Comment:
"Unlike other dimensions that allow parallelization, vectorization, or tensorization (see [Section 7]...), none of these operations can be applied to the fusing dimension. The fusing dimension must be executed sequentially. This constraint enables the safety guarantee discussed below."

Reply:
Excellent rephrasing!